### PR TITLE
Add dedicated KRaft page summarizing various KRaft related information

### PR DESCRIPTION
**Pull request by scholzj**
This PR adds a new page dedicated to various information related to KRaft. The page is not directly linked anywhere. But it could be used for example in presentations or discussions to easily link to the blog posts, docs or videos. The preview of the page can be found here: https://deploy-preview-450--strimzi.netlify.app/kraft/

This PR also updates the list of presentations and conferences with some new talks.

### CODE REVIEW

**Review by im-konge — APPROVED** [review]

**Comment by ppatierno** [comment]
@scholzj sorry Jakub I am struggling to find the point in the website where you can reach the page here https://deploy-preview-450--strimzi.netlify.app/kraft/ AFAIU it should be a band on the main page? But I can't see it.

**Comment by scholzj** [comment]
@ppatierno It is not linked anywhere right now. But you can use it to share it. E.g. in a conference talk where you have 3 minutes to cover KRaft, you can easily link there so that people who are interested can find more details.

**Comment by ppatierno** [comment]
Could it be useful to have a band for some time on the main page like we have for conferences? But I am not sure about it either. Maybe not the right place. Finding a good place on the website would be useful imho.

**Review by ppatierno — APPROVED** [review]

File: _includes/kraft/kraft-title-band.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="secondary-page-title-band">
+  <div class="pattern-wrapper-light">
+    <div class="component-slim grid-wrapper">
+      <div class="width-12-12">
+        <h1>KRaft / ZooKeeper removal</h1>
+        <h3>List of information sources about KRaft support / ZooKeeper removal</h3>
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-title-band.html:6]:
```suggestion
        <h1>KRaft adoption and ZooKeeper removal</h1>
        <h3>Information related to the transition to KRaft</h3>
```

File: _includes/kraft/kraft-content.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<div class="kraft-band">
+  <div class="component-slim grid-wrapper">
+    <div class="width-12-12">
+      <h2 class="blue" id="kraft-information-sources">KRaft information sources</h2>
+      
+      <p>This page lists the various information sources related to KRaft and ZooKeeper removal.
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:6]:
```suggestion
      
      <p>This page lists information sources related to KRaft adoption and ZooKeeper removal.
```

File: _includes/kraft/kraft-content.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <li><a href="https://strimzi.io/docs/operators/latest/deploying#assembly-kraft-mode-str">Using Kafka in KRaft mode</a></li>
+        <li><a href="https://strimzi.io/docs/operators/latest/deploying#deploying-kafka-cluster-kraft-str">Deploying a Kafka cluster in KRaft mode</a></li>
+        <li><a href="https://strimzi.io/docs/operators/latest/deploying#proc-deploy-migrate-kraft-str">Migrating to KRaft mode</a></li>
+      </ul>
+
+      <p></p>
+
+      <h3 id="kraft-blog-posts">Blog posts</h3>
+
+      <p>The following blog posts cover KRaft and ZooKeeper removal:</p>
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:24]:
```suggestion
      <p>The following blog posts cover KRaft adoption and ZooKeeper removal:</p>
```

File: _includes/kraft/kraft-content.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <li><a href="https://strimzi.io/blog/2024/03/21/kraft-migration/">From ZooKeeper to KRaft: How the Kafka migration works</a> (March 21st, 2024)</li>
+        <li><a href="https://strimzi.io/blog/2023/11/02/unidirectional-topic-operator/">Introducing the Unidirectional Topic Operator</a> (November 2nd, 2023)</li>
+        <li><a href="https://strimzi.io/blog/2023/09/11/kafka-node-pools-supporting-kraft/">Kafka Node Pools: Supporting KRaft (ZooKeeper-less Apache Kafka)</a> (September 11th, 2023)</li>
+      </ul>
+
+      <p></p>
+
+      <h3 id="kraft-videos">Videos & Conferences</h3>
+
+      <p>The following videos and conference talks cover KRaft and ZooKeeper removal:</p>
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:38]:
```suggestion
      <p>The following videos and conference talks cover KRaft adoption and ZooKeeper removal:</p>
```

File: _includes/kraft/kraft-content.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<div class="kraft-band">
+  <div class="component-slim grid-wrapper">
+    <div class="width-12-12">
+      <h2 class="blue" id="kraft-information-sources">KRaft information sources</h2>
+      
+      <p>This page lists the various information sources related to KRaft and ZooKeeper removal.
+        Please keep in mind that some of the information in the old videos and blog posts migth be out of date.
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:7]:
```suggestion
        Please keep in mind that some of the information in the old videos and blog posts might be out of date.
```

File: _includes/kraft/kraft-content.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        You should always check the documentation for your Strimzi version that should be always up-to-date.</p>
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:8]:
```suggestion
        Always refer to the documentation for your Strimzi version, which is regularly updated to provide the latest information.</p>
```

File: _includes/kraft/kraft-content.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+      <h3 id="kraft-documentation">Documentation</h3>
+      
+      <p>The following documentation sections cover KRaft, its use and migration of ZooKeeper-based clusters to KRaft:</p>
```
Comment (PaulRMellor) [inline _includes/kraft/kraft-content.html:12]:
```suggestion
      <p>The following documentation sections cover using KRaft and migration from ZooKeeper-based clusters:</p>
```

**Review by PaulRMellor — APPROVED** [review]
good to see all this brought together.
Few minor suggestions

**Review by see-quick — APPROVED** [review]

**Pull request merged**